v2.1/detailed-sql-support.md

```diff
@@ -7,682 +7,682 @@ This page lists which SQL standard features are supported, partially-supported,
 
 To understand the extent to which we support the standard SQL features, use the table below.
 
-- **Feature ID** is the SQL Standard feature identification number.
-- **Description** provides details about the feature.
-- **CockroachDB Support** indicates whether the feature is supported, unsupported, or partially-supported by CockroachDB.
+- Feature ID is the SQL Standard feature identification number.
+- Description provides details about the feature.
+- CockroachDB Support indicates whether the feature is supported, unsupported, or partially-supported by CockroachDB.
```

Comment:
I think the bold in this part is necessary to indicate that we are referring to column headers in the table below.  The original issue only mentions "remove bold formatting from table rows".